### DISCUSSION TITLE
pull request achievement number

### DISCUSSION THREAD

**Discussion by chakravala** [community/community#18380] [Other Feature Feedback, Questions, & Ideas]
if you're going to use powers of 2 to increment the "level" of a user's pull request achievement, then please use powers of 2 properly:
```
2^0     1
2^1     2
2^2     4
2^3     8
2^4    16
2^5    32
2^6    64
2^7   128
2^8   256
2^9   512
2^10 1024
```
level 0 badge should be 1 merged pull request, level 1 is two merged, level 2 is 4 merged, and so on, this is more meaningful / realistic

**Comment by cervonwong** [comment]
There are only four tiers per achievement for simplicity. I assume that's why they skipped some of the powers of 2 in between and used the scale 2, 16, 128, and 1024 instead.

**Comment by melvincwng** [comment]
@chakravala, for "Pull Shark" achievement badge, it goes by 2 PRs, 16 PRs, 128 PRs, 1024 PRs.

Essentially, 2^1, 2^4, 2^7, 2^10.

Hope this clarifies

Reference: https://github.com/Schweinepriester/github-profile-achievements

**Comment by chakravala** [comment]
No, it doesn't clarify anything, in fact the 4 tiers are less useful than
what I proposed. If you're going to make tiers, they need to at least be
useful and not some arbitrary garbage like 16, 128, and 1024.

**Reply by cervonwong** [reply]
It is not arbitrary and don't be too quick to call it garbage. Having just 4 tiers might make each tier more significant and distinctive. And the jump from 16 to 128 to 1024 makes it harder for a person to get to the next tier, making it more impressive.

Though, your suggestion of having powers of 2 might also work too, but it's just not what GitHub chose to do.

**Comment by chakravala** [comment]
No, the 4 tiers are not impressive, they are garbage and not useful for
gauging the relative amount of pull requests. I would be more impressed if
the achievement actually read as meaningful information instead of
basically categorizing everyone the same. As of right now, this is very
unimpressive piece of data, but it might be impressive if it was properly
implemented.
